crates/miden-tx/src/executor/mod.rs

```diff
@@ -285,9 +287,7 @@ where
     // CHECK CONSUMABILITY
     // ============================================================================================
 
-    /// Executes the transaction with specified notes, returning the [NoteAccountExecution::Success]
-    /// if all notes has been consumed successfully and [NoteAccountExecution::Failure] if some note
-    /// returned an error.
+    /// Executes the transaction with specified notes.
```

Comment:
I would expand this comment to provide a bit more info about what this method does.